# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/esden/povray31/man.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./source/unix/makefile.orig	Thu Sep 12 05:12:26 2002
+++ ./source/unix/makefile	Thu Sep 12 05:14:03 2002
@@ -1591,7 +1591,7 @@
 	-@$(RM) $(ODIR)/svga$(OBJ) $(UTARGET) $(XTARGET) $(STARGET)
 
 install:
-	-@cp povray.1 $(POVPATH)/man/man1
+	-@cp povray.1 $(POVPATH)/man/man1/povray31.1
 	-@if \
 	[ -x $(UTARGET) ]; \
 	then \
--- ./source/unix/povray.1~	Mon Nov 22 02:08:53 1999
+++ ./source/unix/povray.1	Thu Sep 12 05:49:49 2002
@@ -3,26 +3,26 @@
 .\"
 
 .SH NAME
-povray, x\-povray, s\-povray \- POV\-Ray: The Persistence of Vision Ray Tracer 
+povray31, x\-povray31, s\-povray31 \- POV\-Ray: The Persistence of Vision Ray Tracer 
 
 .SH SYNOPSIS
-\fBpovray\fP [\fB+i\fP\fIinput_file\fP] [\fB+o\fP\fIoutput_file\fP]
+\fBpovray31\fP [\fB+i\fP\fIinput_file\fP] [\fB+o\fP\fIoutput_file\fP]
 [\fB+/\-option\fP...] [\fIINI_file\fP ...]
 .LP
-\fBx\-povray\fP [\fB+i\fP\fIinput_file\fP] [\fB+o\fP\fIoutput_file\fP]
+\fBx\-povray31\fP [\fB+i\fP\fIinput_file\fP] [\fB+o\fP\fIoutput_file\fP]
 [\fB+/\-option\fP...] [\fIINI_file\fP ...]
 .LP
-\fBs\-povray\fP [\fB+i\fP\fIinput_file\fP] [\fB+o\fP\fIoutput_file\fP]
+\fBs\-povray31\fP [\fB+i\fP\fIinput_file\fP] [\fB+o\fP\fIoutput_file\fP]
 [\fB+/\-option\fP...] [\fIINI_file\fP ...]
 
 
 .SH DESCRIPTION
 \fBPOV\-Ray\fP is a free, full\-featured ray tracer, written and maintained
 by a team of volunteers on the Internet.  On the Unix platform \fBPOV\-Ray\fP 
-can be either a text\-mode only program (\fBpovray\fP), or can optionally 
-have preview capabilities using the \fIX Window System\fP (\fBx\-povray\fP).  
+can be either a text\-mode only program (\fBpovray31\fP), or can optionally 
+have preview capabilities using the \fIX Window System\fP (\fBx\-povray31\fP).  
 Under Linux, \fBPOV\-Ray\fP can optionally use the SVGA library 
-(\fBs\-povray\fP) to preview renderings.
+(\fBs\-povray31\fP) to preview renderings.
 .LP
 This manual page only lists the basic \fBPOV\-Ray\fP and Unix specific
 features and command\-line options for this version of \fBPOV\-Ray\fP.
